pallets/reward-distribution/src/lib.rs

```diff
@@ -70,7 +112,25 @@ pub mod pallet {
 	}
 
 	#[pallet::error]
-	pub enum Error<T> {}
+	pub enum Error<T> {
+		//Overflow
+		Overflow,
+		//underflow
+		Underflow,
+		//if origin tries to withdraw with 0 rewards
```

Comment:
Let's replace these with proper doc comments (`///`) because these will be included in the generated metadata.